### PR TITLE
Cleanup, consistent font sizes

### PR DESCRIPTION
**Pull request by jfrankl**
## Overview

This commit provides general style cleanup for the new application layout. In addition, it attempts to standardize font sizes across the application.

Connects #XXX

### Demo

image

image

image

## Testing Instructions

 * Try to look at every part of the application to see if any of the style is busted

### CODE REVIEW

File: src/mmw/sass/components/_tabs.scss [hunk 1]
```diff
@@ -63,29 +63,29 @@
...
     }
 
     li > a {
       color: $ui-secondary;
-      padding: 0 0.85rem;
-      line-height: 34px;
+      padding: 0 0.7rem;
+      line-height: 42px;
       background-color: transparent;
-      height: $sidebar-tab-subheader-height;
```
Comment (arottersman) [inline src/mmw/sass/components/_tabs.scss]: Maybe we should rename this to `$analyze-tab-height` now that it's only used in the analyze stage.

File: src/mmw/sass/components/_tabs.scss [hunk 2]
```diff
@@ -184,25 +184,30 @@
...
         border: 0;
-        font-weight: 600;
-        background-color: #389b9b !important;
+        font-weight: 800;
         color: $paper;
     }
     .nav-tabs > li {
-        width: 50%;
-        background-color: $ui-secondary;
+        // width: 50%;
```
Comment (arottersman) [inline src/mmw/sass/components/_tabs.scss]: Consider removing.

File: src/mmw/sass/pages/_draw.scss [hunk 3]
```diff
@@ -163,3 +162,47 @@
...
+}
+
+.dropdown-footer {
+  h2 {
+    font-size: $font-size-h3;
+    margin: 0.4rem 0 0.8rem;
+  }
+}
+
+.leaflet-popup-content {
```
Comment (arottersman) [inline src/mmw/sass/pages/_draw.scss]: Consider moving this out of `pages/_draw.scss` and into `base/_map.scss` (where all the leaflet overrides happen) or `pages/_model.scss` (the page it's used on).

File: src/mmw/js/src/core/templates/modificationPopup.html [hunk 4]
```diff
@@ -11,6 +11,6 @@
 </table>
 <span>
     <button class="btn btn-sm btn-icon dark delete-modification" type="button">
-        <i class="fa fa-trash"></i>
+        <i class="fa fa-trash"></i> Remove modification
```
Comment (arottersman) [inline src/mmw/js/src/core/templates/modificationPopup.html]: Consider adding the same [here](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/modeling/templates/gwlfeScenarioToolbar.html#L43)

screen shot 2017-05-25 at 5 05 31 pm

**Comment by arottersman** [comment]
Sorry, didn't bundle this properly. Compare view and model header are actually fine! 

Did notice another issue, however:

New width looks great but cuts off land cover text on chrome.
screen shot 2017-05-26 at 11 20 20 am

Also on chrome noticing some space between the popup tails and body:
screen shot 2017-05-26 at 11 19 42 am

**Comment by jfrankl** [comment]
Fixed the overlapping text:

image

I can't recreate that popup issue. I'm going to merge this as is. If you keep noticing that, please create a new issue.

**Comment by arottersman** [comment]
Thanks! Forgot to hit approve, but tested on FF, Safari, Chrome and IE11 and its 💯

**Pull request merged**